python/tk_desktop2/websockets/requests/commands.py

```diff
@@ -23,7 +23,7 @@ def get_supported_commands():
     from .toolkit_actions import GetActionsWebsocketsRequest
     from .sgc_actions import OpenTaskInSGCreateWebsocketsRequest
     from .sgc_actions import OpenTaskBoardInSGCreateWebsocketsRequest
-    from .sgc_actions import SetMediaPathInSGCreateWebsocketsRequest
+    from .sgc_actions import OpenVersionDraftInSGCreateWebsocketsRequest
```

Comment:
So if I understand correctly, you're saying in that PR description that `SetMediaPathInSGCreateWebsocketsRequest` has just been introduced and was never released to the public, so we don't have to worry about breaking something?

Reply:
Exact